knowhere/index/vector_index/IndexDiskANNConfig.h

```diff
@@ -72,6 +72,8 @@ struct DiskANNPrepareConfig {
     // cached the nodes on the search paths; 2. do bfs from the entry point and cache them. The first method is suitable
     // for TopK query heavy circumstances and the second one performed better in range search.
     bool use_bfs_cache;
+    // The numebr of maximum parallel disk reads per thread. It should be set linearly proportional to `beam_width`.
```

Comment:
Plz add more info here:
1. suggested ratio of this and beam_width
2. what is the limit of total aio for a machine
3. how to modify the max cap.